assets/queries/cloudFormation/aws/s3_bucket_allows_public_acl/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "48f100d9-f499-4c6d-b2b8-deafe47ffb26",
+  "queryName": "S3 Bucket Allows Public ACL",
+  "severity": "MEDIUM",
+  "category": "Access Control",
+  "descriptionText": "S3 bucket allows public ACL",
```

```suggestion
"descriptionText": "S3 bucket should not allow public ACL",
```

I think we should talk with AppSec about this query in terms of severity and category. Additionally to that, I think we should merge "S3 Bucket Allows Public ACL", "S3 Bucket Allows Public Policy", "S3 Bucket Without Ignore Public ACL", and "S3 Bucket Without Restriction Of Public Bucket"